app/policies/accountable_policy.rb

```diff
@@ -1,10 +1,15 @@
 class AccountablePolicy < ApplicationPolicy
+  def initialize(token, resource, scope = :account)
```

Comment:
Does this default `:account` scope get used anywhere?

Reply:
Yes, for all of the models that just use `AccountablePolicy` as their policy class